.github/workflows/integration-tests.yml

```diff
@@ -340,12 +340,12 @@ jobs:
             file: ocr
             pyroscope_env: ci-smoke-ocr-evm-simulated
           - name: ocr2
-            nodes: 4
```

Comment:
@kalverra Do we merge this tests optimisation into develop, or was this just for debugging?

Reply:
I was told I could leave it, just to change the OS back.